penney/functions.py

```diff
@@ -356,6 +356,23 @@ def print_result(scores, seq_dict):
     else:
         print("Tie!")
 
+def print_seq(seq, delay=0.3):
+    """
+    Print a sequence one by one.
+
+    :param seq: round sequence
+    :type seq: str
+    :param delay: delay between each step
+    :type delay: float
+    :return: None
+    """
+    end_str = ""
+    seq_len = len(seq)
+    for index,item in enumerate(seq):
+        if index == (seq_len - 1):
+            end_str = "\n"
+        print(item,end=end_str,flush=True)
```

Comment:
Please add space after `,`, here

Reply:
Thanks for your effort 👍 
fixed in https://github.com/sepandhaghighi/penney/pull/27/commits/071681e137e121f82ff819894b8e6635f5dba9fe